### PR TITLE
last column width not fixed

### PR DESCRIPTION
**Pull request by shaharyar-shamshi**

### CODE REVIEW

File: plugins/compliance-hub/frontend/public/templates/userConsentHistory.html [hunk 1]
```diff
@@ -81,7 +81,7 @@
                     </template>
 
                 </el-table-column>
-                <el-table-column width="230" sortable="custom" prop="ts" label="TIME">
```
Comment (kanwarujjaval) [inline plugins/compliance-hub/frontend/public/templates/userConsentHistory.html:84]: Instead of removing you could make this min-width @shaharyar-shamshi

Comment (itsiprikshit) [reply plugins/compliance-hub/frontend/public/templates/userConsentHistory.html:84]: yeah that would be better.

Comment (shaharyar-shamshi) [reply plugins/compliance-hub/frontend/public/templates/userConsentHistory.html:84]: done

Comment (shaharyar-shamshi) [reply plugins/compliance-hub/frontend/public/templates/userConsentHistory.html:84]: @itsiprikshit  @kanwarujjaval

**Pull request merged**